packages/support/resources/views/components/input/wrapper.blade.php

```diff
@@ -174,11 +175,32 @@
                 />
             @endif
 
-            @if (count($suffixActions))
+            @if (count($suffixActions) || $revealable)
                 <div class="{{ $actionsClasses }}">
-                    @foreach ($suffixActions as $suffixAction)
-                        {{ $suffixAction }}
-                    @endforeach
+                    @if ($revealable)
+                        <x-filament::icon-button
```

Comment:
@danharrin, what do you think about moving these to action methods like we have for repeater/builder?

Reply:
How? That would only be possible if we could trigger JS code from the suffix action, right?

Reply:
We trigger JS code from actions for collapse/expand on repeaters etc

Reply:
Yeah I know. Could you handle reworking the back-end side of this PR then?

Reply:
Yeah I will do